README.md

```diff
@@ -1,45 +1,221 @@
+        <li><a href="#2️⃣-development-phase">2️⃣ Development Phase</a></li>
+        <li><a href="#3️⃣-documentation-phase">3️⃣ Documentation Phase</a></li>
+        <li><a href="#4️⃣-pre-release-phase">4️⃣ Pre-release Phase</a></li>
+        <li><a href="#5️⃣-release">5️⃣ Release</a></li>
+        <li><a href="#6️⃣-post-release">6️⃣ Post-release</a></li>
+      </ul>
+    </li>
+    <li><a href="#📄-license">📄 License</a></li>
+    <li><a href="#🤝-contributing">🤝 Contributing</a></li>
+    <li><a href="#❤️-thanks-to-all-contributors">❤️ Contributors</a></li>
+    <li><a href="#🌟-acknowledgements">🌟 Acknowledgements</a></li>
+    <li><a href="#📰-social-media">📰 Social Media</a></li>
+  </ol>
+</details>
+
+___
+
+<br/>
+
+## 📚 About The Project
+
+<details>
+<summary>
+
+### <b>What is a Decentralized Knowledge Graph?</b>
+</summary>
+
+<br/>
+
+<div align="center">
+    <img src="images/nodes.png" alt="Knowledge Asset" width="200">
+</div>
 
-**Python3 library for interaction with the OriginTrail Decentralized Knowledge Graph**
+There are many available definitions of a knowledge graph, therefore we will present a simplified one focused on usability, rather than completeness. The purpose of this introduction is not to be a comprehensive guide for knowledge graphs, however it aims to get you started with the basics.
 
-The official OriginTrail documentation for v6 can be found [here](https://docs.origintrail.io/dkg-v6-upcoming-version/introduction-to-dkg-v6-start-here).
+A **Knowledge Graph (KG)** is a complex structure that maps the connections and relationships among various entities - be they physical, digital, event-based, or conceptual. Commonly represented as semantic networks, these graphs are leveraged by major corporations such as [Amazon](http://lunadong.com/talks/PG.pdf), [Google](https://en.wikipedia.org/wiki/Google_Knowledge_Graph), [Uber](https://www.youtube.com/watch?v=r3yMSl5NB_Q), and [IBM](https://www.ibm.com/cloud/learn/knowledge-graph) for a wide array of applications, including search, data integration, knowledge reasoning, recommendation engines, analytics, and diverse facets of machine learning and artificial intelligence.
 
+Key characteristics of knowledge graphs include:
 
-## Intro - What is a Decentralized Knowledge Graph (DKG)
+- Prioritization of data connections, treating them as "first-class citizens" within the realm of linked data.
+- Designed to accommodate data from diverse sources, supporting a variety of formats.
+- Their data models are flexible and easily extendable, ready to adapt to the evolving nature of data and its relationships.
 
+While knowledge graphs are typically deployed within the boundaries of a single organization to capture knowledge from various internal and external sources, we conceptualize them more broadly. We define the **Decentralized Knowledge Graph (DKG)** as a globally-shared, common infrastructure for data exchange, benefiting both organizations and individuals.
```

Comment:
I am not sure about this definition, when we google definition for DKG first thing is from our docs, and it sounds better `a global, open data structure comprised of interlinked knowledge assets.`

Reply:
Fixed